website/templates/index.html

```diff
@@ -26,7 +26,7 @@ <h1 class="page-header">{% trans "Find Bugs, Win Points and Prizes" %}</h1>
     <div class="container">
         <center>
             <div class="row">
-                <div class="col-sm-4">    
+                 <div class="col-sm-3 cborder">    
```

Comment:
We can target sm-4 but if we need it somewhere else without border then it won't be possible. 

We shouldn't change the bootstrap classes I think.